ci/gpu/build.sh

```diff
@@ -50,10 +50,8 @@ g++ --version
 conda config --set ssl_verify False
 
 conda install \
-    'cudf=0.8' \
-    'dask-cudf=0.8'
-
-pip install git+https://github.com/dask/distributed.git@master
+    'cudf=0.9' \
+    'dask-cudf=0.9'
```

Comment:
Hard coding these will only end up in causing issues in the future. I believe the better approach is to use what cuDF does from lines [28-31](https://github.com/rapidsai/cudf/blob/branch-0.9/ci/gpu/build.sh#L28). With the `MINOR_VERSION` env var set they you can install the correct versions of cudf/dask-cudf based on your current version.

Reply:
Thanks for pointing that out, I'm not sure why they were hardcoded. This should be resolved in the newest commit.